src/data/polynomial.lean

```diff
@@ -313,6 +313,22 @@ by rw [← C_1, eval₂_C, map_one f]
 instance eval₂.is_add_monoid_hom : is_add_monoid_hom (eval₂ f x) :=
 { map_zero := eval₂_zero _ _, map_add := λ _ _, eval₂_add _ _ }
 
+lemma eval₂_smul (g : R →+* S) (p : polynomial R) (x : S) {s : R} :
+  eval₂ g x (s • p) = g s * eval₂ g x p :=
```

Comment:
There should be a shorter proof of this just by writing `s • p` as `C s * p`, and using `eval₂_mul` and `eval₂_C`